prow/git/v2/executor_test.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package git
+
+import (
+	"bytes"
+	"errors"
+	"github.com/sirupsen/logrus"
+	"reflect"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/util/diff"
+)
+
+func TestCensoringExecutor_Run(t *testing.T) {
+	var testCases = []struct {
+		name        string
+		dir, git    string
+		args        []string
+		censor      func(content []byte) []byte
+		executeOut  []byte
+		executeErr  error
+		expectedOut []byte
+		expectedErr bool
+	}{
+		{
+			name: "happy path with nothing to censor returns all output",
+			dir:  "/somewhere/repo",
+			git:  "/usr/bin/git",
```

Comment:
Nit: When you add something like the `git` binary to each test case I expect a testcase that does have it set differently. Maybe just remove here and hardcode to `/usr/bin/git` for these tests?